app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,24 @@
+require 'net/http'
+
+class VotesController < WelcomeController
+  def create
+    @vote = Vote.new
+    @vote.date = Date.today.beginning_of_day
+    @vote.rating = vote_params[:text].downcase
+
```

Comment:
I would be tempted to save the voting text as the user typed it. Perhaps call it `user_text` and the rating could then be interpreted from that.  So good or +1 could map to Integer `1` and bad, -1, could map to Integer `-1`. In this way calculating the score is a simple `sum`. So `rating` type would be changed to a numerical type